src/Model/Resolver/Products/DataPostProcessor/Attributes.php

```diff
@@ -290,7 +304,7 @@ protected function isAttributeSkipped(
          * On PLP, KEEP attribute if it is used on PLP.
          * This means if not visible on PLP we should SKIP it.
          */
-        return !$attribute->getUsedInProductListing();
+        return !$attribute->getUsedInProductListing() || !$attribute->getStoreLabel();
```

Comment:
Add comment why are you skipping attrs without the label, it is not obvious.

Reply:
Added comment